states/state.py

```diff
@@ -5,21 +5,28 @@
 
 from config.config import TEXT
 
-T = TypeVar('T')
+T = TypeVar("T")
 
 
 class StateOptions(IntEnum):
+    """Base enum class for possible actions in game state"""
+
     def __repr__(self) -> str:
         return f"{self.value} - {self.name}"
 
     @classmethod
     def list_options(cls: Type[T]) -> List[str]:
+        """Returns the options as a string for application to print to user"""
```

Comment:
Minor: `list of strings`, since the return type is not a single string

Reply:
Changed to `Returns the options as a list of strings for application to print to user`